app/renderer/components/bookmarks/bookmarksToolbar.js

```diff
@@ -283,11 +284,23 @@ class BookmarksToolbar extends React.Component {
 
   render () {
     this.bookmarkRefs = []
-    return <div className={css(
-      styles.bookmarksToolbar,
-      this.props.shouldAllowWindowDrag && styles.bookmarksToolbar_allowDragging,
-      !this.props.shouldAllowWindowDrag && styles.bookmarksToolbar_disallowDragging
-    )}
+    return <div
+      className={
+        css(
+          styles.bookmarksToolbar,
+          this.props.textOnly && styles.bookmarksToolbar_textOnly,
+          this.props.shouldAllowWindowDrag && styles.bookmarksToolbar_allowDragging,
+          !this.props.shouldAllowWindowDrag && styles.bookmarksToolbar_disallowDragging
+        ) +
+        // Ensure we do not remove overflow indicator on props change
+        // Aphrodite does not support data-attribute selectors :-(
+        // which would be nice to control this functionality.
+        // Instead, we must use a custom class, added by `calculateNonFirstRowItems`
+        // which gets overriden by this `className` attribute here.
+        (this.hasHiddenKeys
+          ? ' ' + css(styles.bookmarksToolbar_hasOverflow)
+          : '')
```

Comment:
**nit** can't we put it inside `style` instead?

```
style={{ 
  '--bookmarks-toolbar-overflow-indicator-visibility': this.hasHiddenKeys && 'visible',
  '--bookmarks-toolbar-overflow-indicator-width': this.hasHiddenKeys && `${globalStyles.spacing.bookmarksToolbarOverflowButtonWidth} !important`
}}
```

it wouldn't parse if result is `false` and IMO it's easier to read. Again this is **nit**